osslsign.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+IN_FILE=$1
+CERT_PATH=$2
+CERT_PASS=$3
+
+osslsigncode sign -pkcs12 $CERT_PATH -pass $CERT_PASS -in $IN_FILE -out $IN_FILE.sha1  -h sha1 -t http://timestamp.verisign.com/scripts/timstamp.dll
```

Comment:
Can you add `|| exit 1` to the end of the two `osslsigncode` lines and the `mv` line so if any of them fail for some reason, the script will exit? Otherwise it will just keep running and its exit code will be the exit code of the `rm` command, which could be a false positive.